app/src/main/java/com/practicum/playlistmakerapp/presentation/MediaPlayerPresenterImpl.kt

```diff
@@ -0,0 +1,90 @@
+package com.practicum.playlistmakerapp.presentation
+
+import android.os.Handler
+import android.os.Looper
+import com.practicum.playlistmakerapp.domain.interactors.MediaPlayerInteractor
+import com.practicum.playlistmakerapp.domain.models.PlayerState
+import java.text.SimpleDateFormat
+import java.util.*
+
+class MediaPlayerPresenterImpl(
```

Comment:
🍏 Обычно для презентеров не создают интерфейс, поэтому не нужна приставка Impl в реализации.
```suggestion
class MediaPlayerPresenter(
```